packages/wallets/src/components/FlowProvider/FlowProvider.tsx

```diff
@@ -66,39 +66,32 @@ export const useFlow = () => {
  */
 function FlowProvider<T extends TWalletScreens>({
     children,
+    initialScreenId,
     initialValues,
     screens,
-    screensOrder,
 }: TFlowProviderProps<T>) {
-    const [currentScreenId, setCurrentScreenId] = useState<keyof T>(screensOrder[0]);
+    const [currentScreenId, setCurrentScreenId] = useState<keyof T>(initialScreenId || Object.keys(screens)[0]);
     const switchScreen = (screenId: keyof T) => {
         setCurrentScreenId(screenId);
     };
 
     const FlowProvider = FlowProviderContext.Provider as React.Provider<TFlowProviderContext<T> | null>;
-    const currentScreenIndex = useMemo(() => screensOrder.indexOf(currentScreenId), [currentScreenId, screensOrder]);
-    const isFinalScreen = currentScreenIndex >= screensOrder.length - 1;
-
-    const switchNextScreen = () => {
-        if (!isFinalScreen) {
-            const nextScreenId = screensOrder[currentScreenIndex + 1];
-            switchScreen(nextScreenId);
-        }
```

Comment:
The `screensOrder` prop was removed due to an issue where we can't deterministically determine the initial screen order for some flows. For instance, for verification flow the first screen could be to `idv` or `onfido` or `manual`, and the 2nd screen could be directed to `poaScreen` or `personalDetailsScreen`